application-chartjs-ui/src/main/resources/XWiki/ChartJS/WebHome.xml

```diff
@@ -369,6 +402,8 @@ require(['jquery', 'chart', 'datalabels', 'palette'], function($, Chart, datalab
     getData(canvas).then(function(data) {
       return configureColors(data, options, type);
     }).done(function(data) {
+       return removeZeros(data, options, type);
```

Comment:
If you read this code you may think that the zeros are always removed. What I normally do in this case is to:

* either do the ``IF`` here
* or call the function ``maybeRemoveZeros``